draft-ietf-tls-tls13.md

```diff
@@ -1890,14 +1895,16 @@ Cookies serve two primary purposes:
 When sending a HelloRetryRequest, the server MAY provide a "cookie" extension to the
 client (this is an exception to the usual rule that the only extensions that
 may be sent are those that appear in the ClientHello). When sending the
-new ClientHello, the client MUST echo the value of the extension.
-Clients MUST NOT use cookies in subsequent connections.
+new ClientHello, the client MUST echo the value of the extension and should
```

Comment:
It's not related to interoperability, just best practice.  But I would not oppose SHOULD if there is desire to do that.